<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmployeeListScreen</class>
 <widget class="QDialog" name="EmployeeListScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="bonusButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>40</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>BonusBoard</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="employeeSalaryList">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>41</y>
     <width>241</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="exportButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>310</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>310</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Save Changes</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
